### DISCUSSION TITLE
[Feature] Option to Use Local copies of Photos Instead of Streaming from Server

### DISCUSSION THREAD

**Discussion by amal-rgb** [immich-app/immich#11549] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have been using Immich to back up my photos from my Android phone to my server. However, I noticed that after backing up, the app streams photos from the server instead of using the local copies on my phone. This consumes more data and can be slower, especially when I have a large number of photos.

I would like to request a feature where the Immich app prioritizes using local photos if they are available on the device. This would improve the user experience by reducing data usage and increasing the speed of viewing photos.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#9706
